### PR TITLE
hpc: Never install certain modules from HPC repo

### PR DESCRIPTION
**Pull request by Soulofdestiny**

### CODE REVIEW

**Comment by Soulofdestiny** [comment]
verification run: (though it's failing due to another issue - in the video you can see that openqa-test packages are gone from the list)
http://opeth.suse.de/tests/5455

**Comment by Soulofdestiny** [comment]
@asmorodskyi If you want a more 'valid' proof run, let me know

File: lib/utils.pm [hunk 1]
```diff
@@ -1170,14 +1170,16 @@ with ability to exclude some of them by using C<INSTALL_ALL_EXCEPT> which suppos
...
+    my $grep_str     = "";
+    my $hpc_excludes = "hpc-openqa-tools-devel openqa-ci-tools-devel .*openqa-tests";
+    if (get_var('INSTALL_ALL_EXCEPT') or get_var('HPC')) {
         #spliting space separated list of packages into array to iterate over it
         my @packages_array = split(/ /, get_var('INSTALL_ALL_EXCEPT'));
+        push @packages_array, $hpc_excludes if get_var('HPC');
         $grep_str = '|grep -vE "(' . join('|', @packages_array) . ')$"';
     }
-    my $exec_str = sprintf("zypper se -ur %s -t package | awk '{print \$2}' | sed '1,/|/d' %s | xargs zypper -n in", $repo, $grep_str);
+    my $exec_str = sprintf("zypper se -ur %s -t package | awk '{print \$2}' | sed '1,/|/d' %s | xargs zypper -n in -f", $repo, $grep_str);
```
Comment (asmorodskyi) [inline lib/utils.pm]: why we need this -f ?

Comment (Soulofdestiny) [inline lib/utils.pm]: we don't ;)  leftover from debugging

**Pull request merged**